packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -179,7 +179,7 @@ export function createCloudflareEnvironmentOptions(
 				],
 			},
 		},
-		keepProcessEnv: true,
+		keepProcessEnv: false,
```

Comment:
Do we want to hard code this to false or just not set it at all?
What is the default for this value?
What would happen if the user specifies their own value for this?

Reply:
> Do we want to hard code this to false or just not set it at all? What is the default for this value?

Ah yeah... very good point! 👍  `false` (or more precisely `undefined`/falsy: https://github.com/vitejs/vite/blob/63b82f1e29a00d06a82144fd03ea8d6eff114290/packages/vite/src/node/config.ts#L255-L259) is the default... so we might just remove this altogether...

Setting it to `false` makes it very explicit that we're relying on this behavior... but it does feel unnecessary... I don't have a preference here, we can either keep or remove it, what's your preference? 🙂 

 > What would happen if the user specifies their own value for this?

We don't have the `overrides` field anymore so I am not sure if the user can override this 🤔 (I'd imagine that @jamesopstad experimented with this so he should know).
But even if they were to override it not much would change I think, It would likely break the react code but that's about it. The exact same things (user overrides potentially breaking functionalities) would apply to all the other options we set here by the way.

Reply:
I'm happy with removing it altogether.

Need to have a proper look at how we're providing defaults and if they can be overridden as needed but can do that once this is merged.

Reply:
removed 🙂

Reply:
no my bad, it seems like it does need to be set to `false` 😕